pop-api/src/v0/fungibles/errors.rs

```diff
@@ -89,6 +89,15 @@ pub enum PSP22Error {
 	SafeTransferCheckFailed(String),
 }
 
+impl From<PSP22Error> for u32 {
```

Comment:
This needs the `std` feature if we are unwrapping and panicing

Reply:
Resolved in https://github.com/r0gue-io/pop-node/pull/324/commits/ac65b7e5bc8e470c521bf5020a258a439979f872

Reply:
Can we use `expect("<description>")` instead of unwrap. `unimplemented("<description>")` as well.